src/Admin/OnboardingWizardSubmenuPage.php

```diff
@@ -290,4 +288,21 @@ protected function add_preload_rest_paths() {
 			$this->rest_preloader->add_preloaded_path( $path );
 		}
 	}
+
+	/**
+	 * Determine URL that should be used to close the Onboarding Wizard.
+	 *
+	 * @return string Close link.
+	 */
+	public function get_close_link() {
+		$referer                    = wp_get_referer();
+		$excluded_referer_basenames = [ 'customize.php', 'wp-login.php' ];
```

Comment:
The use of `'customize.php'` is not correct in this context. It is such for the Customizer to prevent the close button from ever causing them to go back to the Customizer. In this context, it would be making sure the user doesn't end up going to `admin.php?page=amp-onboarding-wizard` if that were even possible. Otherwise, this should be just removed:

```suggestion
		$excluded_referer_basenames = [ 'wp-login.php' ];
```

Reply:
Ah right, that makes sense.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/5932/commits/3510a88d50e05e9985d4786289a8fbac2743965e.